### DISCUSSION TITLE
My expo app getting crashed after I updated to expo 50 and react native 0.73.6

### DISCUSSION THREAD

**Discussion by arielShubhams** [storybookjs/storybook#29485] [Help]
### Summary

I have tried all stack overflow and google suggetions but nothing works. 

@dannyhw, could you please help in this issue as I am still getting these issues for more then 3 days and there is nothing.

### Additional information

My package are :
    "expo" : "50.0.21",
    "react-native" : "0.73.6",
    "@storybook/addons": "^6.5.8",
    "@storybook/react-native": "^5.3.25", 
    "@storybook/addon-actions": "^8.3.6",
    "@storybook/addon-docs": "^8.3.6",
    "@storybook/addon-links": "^8.3.6",
    "@storybook/addon-notes": "^5.3.21",
    "@storybook/core": "^8.3.6",

**Comment by dannyhw** [comment]
Can you include the error please

**Reply by dannyhw** [reply]
@arielShubhams need more information in order to help you, probably you need to update to the latest version and use  withStorybook in your metro config

Make sure you check the getting started in the github readme for react native storybook

**Reply by dannyhw** [reply]
also remove `@storybook/addons` thats not a thing anymore I'm pretty sure

and you have a lot of weird versions there, you should have everything at v8

**Reply by arielShubhams** [reply]
> Can you include the error please

@dannyhw, RangeError: Maximum call stack size exceeded. and then app stuck on a place.

**Reply by arielShubhams** [reply]
> @arielShubhams need more information in order to help you, probably you need to update to the latest version and use withStorybook in your metro config
> 
> Make sure you check the getting started in the github readme for react native storybook


@dannyhw, when I use latest version it gives error of module not resolved.


also could you please provide me with the supported package and it's dependency as It was working before but when I upgraded the expo version from 48 to 50 it's getting error. probably this issue is related to version or it's dependencies could you confirm that am I going in a right way or need to change the code as well ?

**Reply by dannyhw** [reply]
I really need to see a reproduction of the problem in order to know whats wrong in your project.

You need to make sure you have withStorybook in metro and you need to align all your storybook versions.

Ideally you want to be on at-least expo 50
